sycl/cmake/modules/FetchUnifiedRuntime.cmake

```diff
@@ -115,14 +115,14 @@ if(SYCL_PI_UR_USE_FETCH_CONTENT)
       CACHE PATH "Path to external '${name}' adapter source dir" FORCE)
   endfunction()
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
+  set(UNIFIED_RUNTIME_REPO "git@git.office.codeplay.com:sycl/unified-runtime.git")
```

Comment:
reset this to `https://github.com/oneapi-src/unified-runtime.git"`.

Reply:
Fixed. Updated to `https://github.com/DBDuncan/unified-runtime.git` for CI testing.